docs/en/observability/synthetics-command-reference.asciidoc

```diff
@@ -102,6 +112,16 @@ A unique id associated with your project.
 It will be used for logically grouping monitors.
 If you used <<elastic-synthetics-init-command, `init` to create a project>>, this is the `<name-of-project>` you specified.
 
+`--yes`::
```

Comment:
Suggest duplicating the changes suggested above ([here](https://github.com/elastic/observability-docs/pull/2246/files#r992685103) and [here](https://github.com/elastic/observability-docs/pull/2246/files#r992684298)) by Vignesh, in the `--yes` section too